dydx_withdrawal/views.py

```diff
@@ -54,18 +54,16 @@ def fast_withdrawal(self, request):
         try:
             withdrawal_ref = DydxWithdrawal()
             withdrawal_data = withdrawal_ref.fast_withdrawal(withdrawal_data)
-            result["message"] = withdrawal_data["data"]["withdrawal"]
+
+            result["message"] = withdrawal_data.get('withdrawal')
```

Comment:
previously on line 57, you had written `withdrawal_data["data"]["withdrawal"]` 
and now you have `withdrawal_data.get('withdrawal')`. 
My doubt is, the value of key "withdrawal" is inside the key "data" and if this is true `withdrawal_data.get('withdrawal')` will fail, unless you're returning withdrawal_data['data'] from `fast_withdrawal()`

Reply:
okay and can withdrawal_data be none ? if yes then withdrawal_data.get will fail. None.get == error.

Reply:
it will not because  it's a fast withdrawal  api so when you call it  for sure you will get response  object and there is an then it will  handle  by expectation